app/javascript/react/screens/App/Plan/PlanActions.js

```diff
@@ -10,11 +10,85 @@ import {
   RESET_PLAN_STATE,
   FETCH_V2V_MIGRATION_TASK_LOG,
   DOWNLOAD_LOG_CLICKED,
-  DOWNLOAD_LOG_COMPLETED
+  DOWNLOAD_LOG_COMPLETED,
+  FETCH_V2V_ANSIBLE_PLAYBOOK_TEMPLATE,
+  FETCH_V2V_ORCHESTRATION_STACK
 } from './PlanConstants';
 
 import { V2V_NOTIFICATION_ADD } from '../common/NotificationList/NotificationConstants';
 
+// *****************************************************************************
+// * FETCH_V2FETCH_V2V_ORCHESTRATION_STACK
+// *****************************************************************************
+const _getOrchestrationStackActionCreator = (url, playbookScheduleType, task) => dispatch => {
+  dispatch({
+    type: DOWNLOAD_LOG_CLICKED,
+    payload: task.id
+  });
+
+  return dispatch({
+    type: FETCH_V2V_ORCHESTRATION_STACK,
+    payload: new Promise((resolve, reject) =>
+      API.get(url)
+        .then(response => {
+          resolve(response);
+          dispatch({
+            type: DOWNLOAD_LOG_COMPLETED,
+            payload: task.id
+          });
+          const playbookLogFileName = `${task.vmName}-${playbookScheduleType}.log`;
+          const file = new File([response.data.stdout], playbookLogFileName, { type: 'text/plain;charset=utf-8' });
+          saveAs(file);
+          const successMsg = sprintf(__('"%s" download successful'), playbookLogFileName);
+          dispatch({
+            type: V2V_NOTIFICATION_ADD,
+            message: successMsg,
+            notificationType: 'success',
+            persistent: true,
+            actionEnabled: false
+          });
+        })
+        .catch(e => {
+          dispatch({
+            type: DOWNLOAD_LOG_COMPLETED,
+            payload: task.id
+          });
+          dispatch({
```

Comment:
I don't believe we need to add this notification, as this is a catch on an `API.get` miq notification system should take over signaling the error should any arise...